_posts/2018-06-19-jenkins-build-monitoring-plugin.md

```diff
@@ -0,0 +1,378 @@
+---
+layout: post
+title: "A build monitoring plugin for Jenkins "
+date: 2018-06-19 08:53:01 -0800
+cover: /assets/images/jenkmagic.png
+excerpt: We recently built a plugin to automatically monitor the health of our
```

Comment:
Looks like this line is breaking the build. Jekyll is having trouble parsing the line breaks in the excerpt. I've opened #58 so that we can catch these kinds of issues earlier next time.

Here is a patch that fixes it; I'm not sure if I can push directly to your fork: [fix-excerpt.patch](https://gist.github.com/chrisvogt/13e11ef3a22231c5de6b9488d5397f23) (`git apply fix-excerpt.patch`).

#### Screenshot

<img width="682" alt="screen shot 2018-06-18 at 6 44 53 pm" src="https://user-images.githubusercontent.com/1934719/41571648-fd6f9fcc-7328-11e8-87c5-317a179d11cf.png">